### DISCUSSION TITLE
Web gallery not loading in Microsoft Edge

### DISCUSSION THREAD

**Discussion by pemartinv2** [unoplatform/uno#9455] [General]
When I try to load the Gallery from the web site, I only see the Uno icon on the screen. Nothing seems to load. I tried a couple of configuration in my browser and it still not working. Maybe there is a setting you know that would block the Uno Gallery from loading?

I have Microsoft Edge 104.

**Comment by jeromelaban** [comment]
Thanks for your feedback! Could you show what's in the browser debugger output?

**Reply by pemartinv2** [reply]
<img width="666" alt="image" src="https://user-images.githubusercontent.com/46466777/183809301-d791e83a-b5a1-4f8b-a491-e888c30cfdfe.png">

**Reply by jeromelaban** [reply]
Thanks for the update. This likely means you're using the [Enhanced Security](https://microsoftedge.github.io/edgevr/posts/Introducing-Enhanced-Security-for-Microsoft-Edge/) mode, which disables WebAssembly altogether. Microsoft is [aware of the issue](https://github.com/dotnet/aspnetcore/issues/42641).
